lightning/src/chain/channelmonitor.rs

```diff
@@ -1875,17 +1839,43 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 		Vec::new()
 	}
 
-	/// Called by SimpleManyChannelMonitor::block_connected, which implements
-	/// ChainListener::block_connected.
-	/// Eventually this should be pub and, roughly, implement ChainListener, however this requires
-	/// &mut self, as well as returns new spendable outputs and outpoints to watch for spending of
-	/// on-chain.
-	fn block_connected<B: Deref, F: Deref, L: Deref>(&mut self, txn_matched: &[&Transaction], height: u32, block_hash: &BlockHash, broadcaster: B, fee_estimator: F, logger: L)-> Vec<(Txid, Vec<TxOut>)>
+	/// Checks if a given transaction matches the funding transaction or spends any watched outputs.
+	fn does_tx_match(&self, tx: &Transaction) -> bool {
```

Comment:
fuction name is ambiguous given the comment. `does_tx_match_funding_or_watched()`?

Reply:
I'm up for renaming, but what part is ambiguous? Would `is_relevant_tx` be sufficient or too vague? I was hoping to opt for a shorter names given this is passed to `filter`.

Reply:
`is_relevant_tx` seems great like an improvement, but feel free to throw this in the bikeshed pile and move on.